deploy/kubernetes/rbac.yaml

```diff
@@ -34,6 +34,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
Hm new rbac is going to require us to bump major versions again.  @jsafrane are we ready for this?

Reply:
We need change e2e test of k8s ?

Reply:
I don't remember any other deprecation that would require version bump. It would be nice to include also v1 of CSINode and VolumeAttachment and thus bumping of min. k8s version.

Reply:
I created https://github.com/kubernetes-csi/external-attacher/issues/212 not to forget to bump major version and go module.

I think we can do 3.0 in this cycle.

Reply:
Thanks

Reply:
@jsafrane are we ready to do 3.0 now? This is one of the items preventing us from adding CSI tests to conformance, as conformance cluster only enables v1 objects.